### PR TITLE
Replace ad-hoc check with DB.Ping

### PR DESCRIPTION
**Pull request by pavelkalinnikov**

### CODE REVIEW

File: storage/mysql/admin_storage.go [hunk 1]
```diff
@@ -98,7 +98,7 @@ func (s *mysqlAdminStorage) ReadWriteTransaction(ctx context.Context, f storage.
 }
 
 func (s *mysqlAdminStorage) CheckDatabaseAccessible(ctx context.Context) error {
-	return checkDatabaseAccessible(ctx, s.db)
+	return s.db.Ping()
```
Comment (pavelkalinnikov) [inline storage/mysql/admin_storage.go]: Probably should use PingContext(ctx) instead. Same below.

**Comment by pavelkalinnikov** [comment]
@Martin2112 Any particular reason why `Ping` was not used before?

**Comment by pavelkalinnikov** [comment]
Oh, I see, you might be checking not only availability but also that the DB has a schema?

**Comment by Martin2112** [comment]
Ping had some odd behaviour in early versions of Go. And yes we were checking that the schema existed though I think I scaled that back.

File: storage/mysql/admin_storage.go (hunk 1 above)
Comment (Martin2112) [reply storage/mysql/admin_storage.go]: If we're going to do this it should be PingContext()

**Comment by pavelkalinnikov** [comment]
PTAL.

**Pull request merged**